engine/CMakeLists.txt

```diff
@@ -572,9 +572,9 @@ IF (POLICY CMP0072)
   CMAKE_POLICY (SET CMP0072 NEW)
 ENDIF ()
 
-# Set the default build type to Release
+# Set the default build type to Debug
 IF (NOT CMAKE_BUILD_TYPE)
-    SET(CMAKE_BUILD_TYPE RelWithDebInfo CACHE STRING "Release, Debug, Profiler" FORCE )
```

Comment:
Why do we want this change as part of this PR?

Reply:
This change is really incidental to the main purpose of the PR. I saw something that looked important to fix, and fixed it. We can move it into a separate PR if you feel it's necessary.